gix-hash/Cargo.toml

```diff
@@ -17,9 +17,9 @@ test = false
 
 [features]
 default = ["sha1"]
-## Support for SHA1 digests.
+## Support for SHA-1 hashes and digests.
```

Comment:
The change to `SHA1` was intentional and based on this comment: https://github.com/GitoxideLabs/gitoxide/pull/2292#issuecomment-3639991397 Though I’m fine with any of the variants. :-)